mne/io/brainvision/brainvision.py

```diff
@@ -469,7 +469,10 @@ def _get_eeg_info(vhdr_fname, eog, misc):
         else:
             name, _, resolution, unit = props[:4]
         ch_names[n - 1] = name
-        cals[n - 1] = float(resolution)
+        try:
+            cals[n - 1] = float(resolution)
+        except ValueError:
+            cals[n - 1] = 0.000001
```

Comment:
can you catch the error with resolution explicitly rather than a try catch? what would be the value of resolution in case of ValueError?